### PR TITLE
Branch2pr

### PR DESCRIPTION
**Pull request by ajose1024**
Small updates:

--> Copyright notice update (from 2021 to 2023)
--> Small text addition in index.html main body
--> CSS class addition to allow for flex direction choice

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,8 +28,10 @@ <h1>Hello Microverse Project</h1>
             </ul>
         </nav>
 
-        <div class="main_body flex-box body-text">
+        <div class="main_body flex-box flex-dir-column body-text">
             <p>Hello to all in Microverse!</p>
```
Comment (divinecharlotte) [inline index.html]: - [ ] Kindly consider changing this p into an `h1 ` as requested in the project requirements to fulfill them

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ajose1024   , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters are passing 👍 
- Descriptive pr description👍 

 




 

## Required Changes ♻️

- [x]  kindly consider giving your Pull Request a descriptive title telling you what changes are inside this current pr to make its title descriptive

- [x]  kindly consider checking out to another branch and make sure that branch is descriptive remember a descriptive Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section') and after doing so consider including the link to this current pr in the new pr description you are going to have to make the review process smooth

- [x] Kindly consider making your repository look more professional by checking the below points 👇 

- README file should be descriptive? AKA: Microverse [README template](https://github.com/microverseinc/readme-template) should be used.


1. All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo.
2. The only optional section is FAQ.
3. you should delete the live demo section if you decide not to deploy the application.
4. Make sure you add the section Future Features, including 1-3 potential features to be added.

- [x] Last but not least kindly consider making sure that all your Linters are passing
```
Linter should be set up correctly for this repository.
Linter result should be green. AKA: Linter should report NO errors.
```
to make sure all your codes are free from errors cause current below is how your Linters are looking
Screen Shot 2023-07-05 at 10 51 37

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ajose1024) [reply index.html]: Ok, that makes all sense to make the use of html tags more semantical

**Comment by ajose1024** [comment]
Suggestions implemented and some found errors corrected.

Found errors:
-> one of the context link-style class selector had a error in the class name. It was corrected
-> some of the context link-style:visited colors had poor contrast with the background. It was changed to more appropriated colors
-> suggestion to replace the first <p> container in the main-body section by a <h1> container was followed and implemented.

**Comment by ajose1024** [comment]
To implement the suggestion of a more meaningful branch name, I will have to close this pull request, change the name of the branch and then open a new pull request.

**Comment by ajose1024** [comment]
To implement the suggestion of a more meaningful branch name, I will have to close this pull request, change the name of the branch and then open a new pull request.

**Pull request closed without merging**